RELEASE/scripts/autoscend/auto_boris.ash

```diff
@@ -106,6 +139,8 @@ boolean boris_buySkills()
 			skillPoints = skillPoints - 1;
 			int tree = 1;
 
+			//skills are listed in reverse order. from last to first to buy.
+			//Correct Boris strat is super easy. get all feasting, then all shouting, then fighting last.
```

Comment:
This could be cleaned up as something like

```
boolean[skill] fighting_skills = $skills[...]
boolean[skill] feasting_skills= $skills[...]
boolean[skill] shouting_skills = $skills[...]

boolean missing_skill(boolean[skill] sks) {
  // returns if you're missing a skill in sk
}

if (missing_skill(feasting)) tree = 3;
else if (missing_skill(shouting)) tree = 2;
else if (missing_skill(fighting)) tree = 1;
```

probably don't do this in this commit, just flagging 120 unnecessary lines

Reply:
Agreed. both on the improvement and on doing it in a future commit.